src/test/java/duke/UiTest.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class UiTest {
+    private final ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+    
+    @Test
+    void numberOfTasks_empty() {
+        Ui ui = new Ui();
+        String output = ui.printNumberOfTasks(0);
+        assertEquals("Now you have 0 task in your list.", output);
+    }
+    
+    @Test
+    void printTask_emptyList() {
+        Ui ui = new Ui();
+        System.setOut(new PrintStream(outputStream));

```

Comment:
I like how you were able to redirect your output stream here for testing purposes!